### DISCUSSION TITLE
Win11 24H2 : combo ExplorerPatcher & OpenShell work well (edit)

### DISCUSSION THREAD

**Discussion by olivier-at-home** [Open-Shell/Open-Shell-Menu#2051] [General]
I am using the combo OpenShell & ExplorerPatcher.

Upgrading Win11 to 24H2 required me to uninstall both OpenShell & ExplorerPatcher.

ExplorerPatcher is documented and supported on 24H2. I reinstalled it, and it's working fine.

What's about OpenShell ?  Can I re-install it safely ? 

Thank you.

**Comment by ge0rdi** [comment]
`Open-Shell` should work fine on 24H2 (just make sure you have latest version).
Though I can't say whether it works fine along with `ExplorerPatcher`.

**Comment by olivier-at-home** [comment]
Thanks. I confirm : OpenShell 4.4.191 + ExplorerPatcher combo work well on Win11 24H2

![image](https://github.com/user-attachments/assets/127c95b3-bad4-47f3-ba67-e398771842b0)

![image](https://github.com/user-attachments/assets/48ac04a3-c88e-4462-8418-072859a2f55d)
